src/js/App/Header/ToolbarToggle.js

```diff
@@ -40,7 +40,7 @@ class ToolbarToggle extends Component {
     const dropdownItems = this.props.dropdownItems.map(({ url, title, onClick, isHidden, target = '_blank', rel = 'noopener noreferrer', ...rest }) =>
       !isHidden ? (
         <DropdownItem
-          key="item"
+          key={title}
```

Comment:
Just a suggestion for the future: add `key` and `oudaId` attributes to `dropdownItems` in case that title is not a string 🤔

Reply:
Yeah, this "should" always be string based on how we use this component. But we should definitely address this in the post-summit refactoring.